.gem_ffox_init.sh

```diff
@@ -0,0 +1,35 @@
+export GEM_FIREFOX_ON_HOLD=
```

Comment:
With this line any value assigned in the CI job will be ignored, remove it, please.